### ISSUE TITLE
[Bug] Timestamp in visualisation (redis insight) seems wrong

### ISSUE THREAD

**Issue by lyrixx** [RedisTimeSeries/RedisTimeSeries#332]
Hello.

I added few data point with redis cli:
```
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380038011
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380038556
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380038795
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380038987
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380039177
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380039363
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380039526
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380039696
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380039899
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380040041
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380040275
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380040509
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380040701
172.21.0.12:6379> TS.ADD test_insight * 1
(integer) 1580380040889
172.21.0.12:6379> TS.ADD test_insight * 1
```

Then I open redis insight and the X axis seems broken

![image](https://user-images.githubusercontent.com/408368/73441705-b23e5f00-4353-11ea-8c6b-b23124977139.png)

> RedisInsight version 1.2.0.

As you can see, all date are wrong.

I'm not sure this should be reported here or somewhere else. If I'm wrong, I'm sorry. Fell free to move the issue to the right repo.

Thanks

**Comment by K-Jo** [comment]
@lyrixx we'll try to push out a new version of RedisInsight this week (1.2.1) which will contain the fix.  Will keep this ticket updated

**Comment by lyrixx** [comment]
@K-Jo Thanks :yellow_heart:

**Comment by K-Jo** [comment]
@lyrixx this has been resolved in v1.2.1 https://docs.redislabs.com/latest/ri/release-notes/v1.2.0/

**Comment by lyrixx** [comment]
Thanks :+1:
